website/source/docs/providers/aws/r/elasticache_cluster.html.markdown

```diff
@@ -73,6 +73,15 @@ names to associate with this cache cluster
 Amazon Resource Name (ARN) of a Redis RDB snapshot file stored in Amazon S3. 
 Example: `arn:aws:s3:::my_bucket/snapshot1.rdb`
 
+* `snapshot_window` - (Optional) The daily time range (in UTC) during which ElastiCache will 
+begin taking a daily snapshot of your cache cluster. Can only be used for the Redis engine. Example: 05:00-09:00
+
+* `snapshow_retention_limit` - (Optional) The number of days for which ElastiCache will 
+retain automatic cache cluster snapshots before deleting them. For example, if you set 
+SnapshotRetentionLimit to 5, then a snapshot that was taken today will be retained for 5 days 
+before being deleted. If the value of SnapshotRetentionLimit is set to zero (0), backups are turned off. 
+Can only be used for the Redis engine.
+
```

Comment:
Another gem: snapshotting is not enabled for `t2` instance types, like the ones I've been testing with (explaining much :laughing: )

> Redis backup/restore is not supported for t2 instances.

- http://docs.aws.amazon.com/fr_fr/AmazonElastiCache/latest/APIReference/API_CreateCacheCluster.html